tools/deploy_ipk.sh

```diff
@@ -40,7 +40,13 @@ pgrep opkg | xargs kill -SIGINT
 opkg remove --force-depends prplmesh prplmesh-dwpal prplmesh-nl80211
 # currently opkg remove does not remove everything from /opt/prplmesh:
 rm -rf /opt/prplmesh
-opkg install -V2 "$DEST_FOLDER/$IPK_FILENAME"
+# rdkb platforms require --force-dependencies
+INSTALL_ARGS="-V2"
+. /etc/os-release >/dev/null 2>&1
+if [ $ID==rdk ]; then INSTALL_ARGS+=" --force-dependencies"; fi
```

Comment:
```suggestion
if [ $ID = rdk ]; then INSTALL_ARGS+=" --force-dependencies"; fi
```
You forgot the space around the equal operator, so you aren't testing anything. Also '==' only makes sense for bash, while '=' has the same behavior for bash and other shells.

Reply:
Good catch!

Reply:
Thank you, fixed